### DISCUSSION TITLE
İs it possible to import database from commandline?

### DISCUSSION THREAD

**Discussion by erickyun** [danrahn/MarkerEditorForPlex#28] [Q&A]
Hi @danrahn ,

I am running MarkerEditorForPlex App with docker compose I was wondering if I could create crontab that calls you App to import database?

e.g. docker exec -it intro-editor-for-plex --import /directory/export-library-3.db

**Comment by danrahn** [comment]
That's not possible right now unfortunately. I've considered an command-line interface, but it would likely require some substantial refactoring, and I'm also _very_ unfamiliar with Docker, so figuring out exactly how `docker exec` works might require some extra work on top of everything else. I'll keep this in mind, but chances are it isn't something I'll get to any time soon, if at all.
